src/PrettierEditProvider.ts

```diff
@@ -31,9 +31,26 @@ const STYLE_PARSERS: ParserOption[] = ['postcss', 'css', 'less', 'scss'];
  * @param filePath file's path
  */
 async function hasPrettierConfig(filePath: string) {
-    return (await bundledPrettier.resolveConfig(filePath)) !== null;
+    const { config } = await resolveConfig(filePath);
+    return config !== null;
 }
 
+type ResolveConfigResult = { config: Partial<PrettierConfig>, error?: Error };
```

Comment:
Happy to "inline" this type and/or move it to another file. This seemed like a simple solution to start though.

Reply:
This type is local to this file, it fits well in there.
Could be an inline type as it is used once ... But it is also good as it is :+1:

Reply:
I also have on my todo list:
Clean/correct our types